MVPSt_2023_Jan/MVPSt_2023_Jan/StepDefinitions/SellerProfileFeatureStepDefinitions.cs

```diff
@@ -1,42 +1,47 @@
 using MVPSt_2023_Jan.Pages;
 using MVPSt_2023_Jan.Utilities;
-using OpenQA.Selenium.Chrome;
 using NUnit.Framework;
 
+
 namespace MVPSt_2023_Jan.StepDefinitions
 {
     [Binding]
-    public class Seller_Profile_FeatureStepDefinitions : CommonDriver
+
+    public class SellerProfileFeatureStepDefinitions : CommonDriver
     {
-        // create page object initializations and definitions
-        TYSPortalPage tysportalPageObj = new TYSPortalPage();
-        HomePage homePageObj = new HomePage();
+        // define Pages and Objects
+        //TYSPortalPage tysportalPageObj;
+        HomePage homePageObj;
+        // default constructor
+        public SellerProfileFeatureStepDefinitions()
+        {
+            //tysportalPageObj = new TYSPortalPage();
+            homePageObj = new HomePage();
+        }
 
         [Given(@"I have signed into Trade Your Skill portal successfully using email ""([^""]*)"" and password ""([^""]*)""")]
         public void GivenIHaveSignedIntoTradeYourSkillPortalSuccessfullyUsingEmailAndPassword(string Email, string Pswd)
         {
-            // open Chrome browser because of using OpenQA.Selenium.Chrome; statement at top of code
-            driver = new ChromeDriver();
-
-            tysportalPageObj.SigninActions(driver);
+            SetupDriver();
```

Comment:
you don't need to do this as the Hook will execute this for you!

Reply:
Removed call to SetupDriver in Given logic in step definition file